pkg/appgw/test_fixtures.go

```diff
@@ -108,7 +124,7 @@ func makeIngressTestFixture() v1beta1.Ingress {
 						HTTP: &v1beta1.HTTPIngressRuleValue{
 							Paths: []v1beta1.HTTPIngressPath{
 								{
-									Path: "/a/b/c/d/e",
+									Path: testFixturesURLPath,
 									Backend: v1beta1.IngressBackend{
 										ServiceName: "",
```

Comment:
serviceName is empty here. Will we populate this in the test ?

Reply:
Added a `ServiceName`